### ISSUE TITLE
[UI] 5.0: scrollbar won't appear when hoover over

### ISSUE THREAD

**Issue by zarnovican** [grafana/grafana#11168]
Please include this information:
- What Grafana version are you using? 5.0.0, 5.0.1
- What datasource are you using? InfluxDB
- What OS are you running grafana on? Ubuntu
- What did you do?

When you load a dashboard that is larger than screen height, horizontal scrollbar is hidden. When you hoover over it with a mouse pointer it will _not_ show up.

- What was the expected result?

Scrollbar should appear when hoovered over, or it should be visible by default when the view is larger than the screen.

- What happened instead?

Scrollbar remains hidden.
It will appear only when you scroll down by keyboard or mouse wheel. After that it will start working as expected.

Different parts of UI are affected by the problem as well. For example selecting from a long list of dashboards is also missing a scrollbar. There, even the keyboard trick won't save you.

Browser:
- Chrome - Version 59.0.3071.115 (Official Build) (64-bit)
- Firefox - 58.0.1 (64-bit)

**Comment by Caffe1neAdd1ct** [comment]
Can also confirm this issue persists across both the light and dark themes. Once the page has been scrolled the bar does normally appear. On the light theme the scroll bars are very hard to see.

**Comment by tw-bert** [comment]
We are also experiencing this, so staying on 4.x for now.

**Comment by Caffe1neAdd1ct** [comment]
Page loaded before scrolling
```
<div class="scroll-canvas scroll-canvas--dashboard ps" grafana-scrollbar=""></div>
<div class="ps__rail-x" style="left: 0px; bottom: 0px;">
    <div class="ps__thumb-x" tabindex="0" style="left: 0px; width: 0px;"></div>
</div>
<div class="ps__rail-y" style="top: 0px; right: 0px;">
    <div class="ps__thumb-y" tabindex="0" style="top: 0px; height: 0px;"></div>
</div>

```

Page after manually scrolling with mouse wheel or keyboard controls:
```
<div class="scroll-canvas scroll-canvas--dashboard ps ps--active-y" grafana-scrollbar="">
<div class="ps__rail-x" style="left: 0px; bottom: 0px;">
    <div class="ps__thumb-x" tabindex="0" style="left: 0px; width: 0px;"></div>
</div>
<div class="ps__rail-y" style="top: 0px; right: 0px; height: 406px;">
    <div class="ps__thumb-y" tabindex="0" style="top: 0px; height: 198px;"></div>
</div>
```

**Comment by Caffe1neAdd1ct** [comment]
Might be related to https://github.com/utatti/perfect-scrollbar/issues/120

**Comment by tw-bert** [comment]
@bergquist : Can you confirm this bug? We'd like to jump on the 5.x wagon, but this issue is preventing us from doing so.

**Comment by torkelo** [comment]
We are using overlay scrollbars , look/work similar to upcoming browser overlay scrollbar native feature (only available in chrome at the moment), overlay scrollbars only show when you start scrolling , and disappear after some seconds after scroll completed, they also show when you hover over the edge.

**Comment by torkelo** [comment]
Ah, sorry read the issue wrong, should appear when you hover.

**Comment by tw-bert** [comment]
@torkelo I just tried this in Grafana `5.0.4`, and this issue is still there. No scrollbar appears on hover in the right side of Grafana. We'd very much appreciate a fix (not all people in our company have/use a scroll wheel, so only the keyboard-arrow-workaround, or using `4.x`, works).

**Comment by torkelo** [comment]
we are working on a fix here, https://github.com/grafana/grafana/pull/11456 

will be included in v5.1 release

**Comment by daniellee** [comment]
Fixed by #11456

**Comment by or4** [comment]
fix not working for me, I fix it by
```
    const [, setNum] = React.useState<number>()
    useEffect(() => {
      // this fix need to show both scrollbars for perfect-scrollbar
      setTimeout(() => {
        setNum(Date.now())
      }, 200)
    }, [setNum])
```
